src/components/Culture/index.js

```diff
@@ -0,0 +1,269 @@
+      </div>
+      <div className="culture-element">
+        <h4>
+          Performing Arts Venues
+          <img
+            data-cy="culture-icon"
+            className="culture-icon"
+            src={theater}
+            alt="Stage to symbolize performing arts venues"
+          />
+        </h4>
+        <p data-cy="venue-value">
+          {city.attributes.details.culture.performing_arts_venues}
+        </p>
+      </div>
+      <div className="culture-element">
+        <h4>
+          Sports Venues
+          <img
+            data-cy="culture-icon"
+            className="culture-icon"
+            src={arena}
+            alt="Arena to symbolize stadiums"
+          />
+        </h4>
+        <p data-cy="venue-value">
+          {city.attributes.details.culture.sports_venues}
+        </p>
+      </div>
+      <div className="culture-element">
+        <h4>
+          Zoos
+          <img
+            data-cy="culture-icon"
+            className="culture-icon"
+            src={zoo}
+            alt="Zoo entrance to symbolize venues"
+          />
+        </h4>
+        <p data-cy="venue-value">{city.attributes.details.culture.zoos}</p>
+      </div>
+    </StyledCulture>
+  ) : (
+    <StyledCulture>
+      <h2>Culture</h2>
+      <div className="culture-element">
+        <h3>{selectedCities[0].attributes.name}</h3>
+        <h3>{selectedCities[1].attributes.name}</h3>
+      </div>
+      <div className="culture-element">
+        <p>{selectedCities[0].attributes.details.culture.art_galleries}</p>
+        <h4>
+          Art Galleries
+          <img
+            className="culture-icon"
+            src={artGallery}
+            alt="Mona Lisa to symbolize art galleries"
+          />
+        </h4>
+        <p>{selectedCities[1].attributes.details.culture.art_galleries}</p>
+      </div>
+      <div className="culture-element">
+        <p>{selectedCities[0].attributes.details.culture.comedy_clubs}</p>
+
+        <h4>
+          Comedy Clubs
+          <img
+            className="culture-icon"
+            src={comedyClub}
+            alt="Microphone to symbolize comedy clubs"
+          />
+        </h4>
+        <p>{selectedCities[1].attributes.details.culture.comedy_clubs}</p>
+      </div>
+      <div className="culture-element">
+        <p>{selectedCities[0].attributes.details.culture.concert_venues}</p>
+
+        <h4>
+          Concert Venues
+          <img
+            className="culture-icon"
+            src={concert}
+            alt="Concert hall to symbolize venues"
+          />
+        </h4>
+        <p>{selectedCities[1].attributes.details.culture.concert_venues}</p>
+      </div>
+      <div className="culture-element">
+        <p>{selectedCities[0].attributes.details.culture.historical_sites}</p>
+
+        <h4>
+          Historical Sites
+          <img
+            className="culture-icon"
+            src={historicSite}
```

Comment:
Love all of these images! These will really pop and give us some design inspiration to match.